site-src/concepts/versioning.md

```diff
@@ -1,116 +1,70 @@
 # Versioning
 
-## Summary
-Each Gateway API release is represented by a bundle version that represents
-that specific combination of CRDs, API versions, and validating webhook. To
-enable experimental fields, future releases of the API will include stable and
-experimental CRD tracks. Users will be able to access experimental features by
-installing the experimental CRDs. A cluster can contain either an experimental
-or stable CRD for any resource at a given time. 
+## Overview
+Each new release of Gateway API is defined with a "bundle version" such as
+v0.4.0. This contains the following:
```

Comment:
if bundle version is the git tag, we should say so.

Reply:
Updated, thanks!